cmd/jaeger/ingester.yaml

```diff
@@ -0,0 +1,32 @@
+service:
```

Comment:
Why do we need this file? Ingester with memory storage doesn't make a lot of sense, since nothing else can access that storage. And you also have jaeger_query enabled, which doesn't fit the meaning of ingester.